src/main/java/com/google/sps/daos/DatabaseRoomDao.java

```diff
@@ -0,0 +1,75 @@
+package com.google.sps.daos;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List; 
+import java.util.logging.Logger;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.firestore.QuerySnapshot;
+import com.google.cloud.firestore.SetOptions;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firestore.v1.StructuredQuery.CollectionSelector;
+
+import com.google.sps.data.Member;
+import com.google.sps.data.Room;
+
+public class DatabaseRoomDao implements RoomDao {
+
+  private DatabaseReference roomsDBReference;
+  private Firestore firestoreDb;
+  private String actualRoom;
```

Comment:
This is not used anymore correct? Also `roomsDBReference`

Reply:
Yess you are right, done!